docs/src/statsplots.md

```diff
@@ -90,3 +90,79 @@ autocorplot(chn)
 ```@example statsplots
 corner(chn)
 ```
+
+For plotting multiple parameters, Ridgeline, Forest and Caterpillar plots can be useful.
+Please see the docstrings for a detailed description of these functions.
+
+## Ridgeline
+
+```@example statsplots
+ridgelineplot(chn, [:C, :B, :A])
+```
+""""
```

Comment:
Docstrings have to be attached to functions and should not be added to the docs directly.